src/main/java/seedu/mtracker/commands/ListCommand.java

```diff
@@ -1,11 +1,21 @@
 package seedu.mtracker.commands;
 
+import seedu.mtracker.instrument.Instrument;
+import java.util.ArrayList;
+
 public class ListCommand extends Command {
 
     public static final String COMMAND_WORD = "list";
+    private static final String SPACE = " Current Price: ";
+
+    private ArrayList<Instrument> instruments;
 
     @Override
     public String execute() {
+        instruments = instrumentManager.getInstruments();
+        for (Instrument i : instruments) {
+            System.out.println(i.toString() + SPACE + i.getCurrentPrice());
+        }

```

Comment:
Shift this functionality into a function that takes in instruments along with SPACE constant into TextUi. Maybe can rename SPACE to like CURRENT_PRICE_HEADER or CURRENT_PRICE_FIELD